core/sql/common/NAString.cpp

```diff
@@ -280,6 +280,29 @@ NAString Int64ToNAString(Int64 l)
   return NAString(resultstr);
 }
 
+NAString &replaceAll(NAString &source, const NAString &searchFor,
+                     const NAString &replaceWith)
+{
+  size_t indexOfReplace = NA_NPOS;
+  indexOfReplace = source.index(searchFor);
+  if (indexOfReplace != NA_NPOS)
```

Comment:
yes, that is true. 
This method was moved from sqlcomp/CmpDescribe.cpp to common dir
and is not new code.
Will fix it in a later checkin.